ompl/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
```

Comment:
add_compile_options was introduced in 2.8.12 it seems

https://cmake.org/cmake/help/v2.8.12/cmake.html#command:add_compile_options